tests/functional/test_updating_manifest.py

```diff
@@ -50,6 +49,10 @@ def test_reading_manifest(tmpdir):
     XML manifest files are written with UTF-8 even localized Windows.
     We check here not to use local encoding against UTF-8 manifest file.
     """
+    # This import only works on Windows. Place it here, protected by the
+    # ``@skipif_notwin`` decorator.
+    from PyInstaller.utils.win32 import winmanifest
+
```

Comment:
Unrelated change slipped in. Looks like you need to rebase :-)

Reply:
Should be fixed already -- perhaps this is github's confusion.